arithmetization/src/main/java/net/consensys/linea/zktracer/module/trm/Trm.java

```diff
@@ -100,9 +100,8 @@ public void commit(List<MappedByteBuffer> buffers) {
     final Trace trace = new Trace(buffers);
 
     int stamp = 0;
-    for (TrmOperation operation : this.operations) {
-      stamp++;
-      operation.trace(trace, stamp);
+    for (TrmOperation operation : operations.getAll()) {
+      operation.trace(trace, ++stamp);
     }
```

Comment:
What is the reason for the `++stamp` rather than `stamp++`. IIRC it's something about updating a value prior / post something else ...

Reply:
Nevermind, ChatGPT helped me with this 🙂